src/apple-other.rs

```diff
@@ -1,4 +1,4 @@
-// Copyright 2023 Developers of the Rand project.
+// Copyright 2023-2024 Developers of the Rand project.
```

Comment:
Maybe we should remove copyright lines from source code files? Personally, I think they are somewhat pointless, especially considering that we do not include full copyright notices as stated by MIT and Apache licenses

Reply:
I can't offer legal advice @newpavlov.

That said, I guess the purpose may be to have a documented source/copyright owner in case a file is copied in absense of the rest of the code. That said, searching "the Rand project" on Google doesn't appear to help (we should have chosen a better name?).

Reply:
I think that including license information in each file is important for project which mix different licenses (like the horrible mess in the pre-3.0 OpenSSL). In our case we require that all contributions are licensed under MIT OR Apache 2.0 terms (could be worth to add an explicit [contribution section](https://github.com/RustCrypto/traits#contribution) to README) and we use the umbrella copyright "owner" instead of recording personal contributions in licensing notices.

We do not distribute separate source files, in our case unit of distribution is the whole crate and we already include the appropriate license files in published crates and keep them as part of the repository.

In the Apache license appendix it's implied that the notice should be included in each source file to apply the license. But in my understanding it's nothing more than a recommendation and, either way, we do not follow the notice format in the first place.

Reply:
Do we lose anything by removing the notice? I don't think so since copyright is supposed to be automatic.

Maybe the best option is to mention the licence with a link to the COPYRIGHT file (if we even have that here) in the README and in the top-level library docs (so that it shows up in generated docs; strange that nobody seems to do that), but not bother in other source files (aside from the Cargo.toml line).

Reply:
I think the current [notice](https://github.com/rust-random/getrandom#license) in README should be sufficient and adding a similar notice to the library docs sounds reasonable. In future, after an appropriate MSRV bump we should use `#![doc = include_str!("../README.md")]` to reduce duplication.

>strange that nobody seems to do that

I think most people simply consider this information redundant in docs. Documentation is for people who want to learn how to use the crate, while licensing information is available in Cargo.toml, on crates.io, and even on docs.rs. Plus, it can be retrieved automatically using tools like `cargo license` and overwhelming amount of crates on crates.io is published under liberal licenses, so having similar MIT/Apache terms in most of crates' docs would feel like a waste of space.